app/controllers/accounts_controller.rb

```diff
@@ -15,7 +15,9 @@ def show
         render xml: AtomSerializer.render(AtomSerializer.new.feed(@account, @entries.to_a))
       end
 
-      format.activitystreams2
+      format.activitystreams2 do
+        headers['Access-Control-Allow-Origin'] = '*'
```

Comment:
You should check `config/application.rb` for CORS settings, and move all of the CORS stuff there.